app/controllers/application_controller.rb

```diff
@@ -1,7 +1,16 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
```

Comment:
this protected section (along with method `configure_permitted_parameters`) should be at the bottom of class, and `include`s would be better to specify in the beginning of class.